### PR TITLE
Switch to TravisCI

### PR DESCRIPTION
**Pull request by snormore**

### CODE REVIEW

**Comment by stephensolis** [comment]
https://github.com/triagemd/ml-tools/blob/master/tests/conftest.py#L23

```
import requests

response = requests.get('https://storage.googleapis.com/tf-serving-docker-http-eae9e0c7-661d-4cca-836a-0433a8da44ba/imagenet/dictionary.csv')
reader = csv.reader(response.text)
```

is going to be a lot easier for python 2 and 3 compatibility

File: setup.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
         'keras',
         'h5py',
         'pillow',
+        'requests',
```
Comment (stephensolis) [inline setup.py]: Can you add this to the requirements.txt instead? It isn't a library requirement

Comment (stephensolis) [inline setup.py]: Can you add this to the requirements.txt instead? It isn't a library dependency

Comment (snormore) [inline setup.py]: Oh yep, nice catch

File: tests/conftest.py [hunk 2]
```diff
@@ -2,8 +2,7 @@
 import csv
 import tempfile
 import codecs
-
-from urllib.request import urlopen
+import requests
 
 
 @pytest.fixture(scope='function')
```
Comment (stephensolis) [inline tests/conftest.py]: `tempfile.TemporaryDirectory()` isn't in Python 2, `tempfile.mkdtemp` might work instead

**Comment by stephensolis** [comment]
> `reader = csv.reader(response.text)`

Oops, actually meant `reader = csv.reader(response.text.split())`

File: tests/conftest.py (hunk 2 above)
Comment (stephensolis) [inline tests/conftest.py]: ditto below `tempfile.NamedTemporaryFile()`, `tempfile.mkstemp`

File: tests/conftest.py [hunk 3]
```diff
@@ -20,7 +19,7 @@ def temp_file():
 
 @pytest.fixture
 def imagenet_dictionary():
-    response = urlopen('https://storage.googleapis.com/tf-serving-docker-http-eae9e0c7-661d-4cca-836a-0433a8da44ba/imagenet/dictionary.csv')
-    reader = csv.reader(codecs.iterdecode(response, 'utf-8'))
+    response = requests.get('https://storage.googleapis.com/tf-serving-docker-http-eae9e0c7-661d-4cca-836a-0433a8da44ba/imagenet/dictionary.csv')
+    reader = csv.reader(response.text)
```
Comment (stephensolis) [inline tests/conftest.py]: Oops, actually meant `reader = csv.reader(response.text.split())`

**Comment by snormore** [comment]
Updated `script/env` to use Python 2 in dev too. We'll follow-up with a PR that tests multiple versions.

File: tests/conftest.py (hunk 2 above)
Comment (snormore) [inline tests/conftest.py]: Ended up using `backports.tempfile` for this

File: tests/conftest.py [hunk 4]
```diff
@@ -1,26 +1,27 @@
...
+    with NamedTemporaryFile() as f:
         yield f.name
 
 
 @pytest.fixture
 def imagenet_dictionary():
-    response = urlopen('https://storage.googleapis.com/tf-serving-docker-http-eae9e0c7-661d-4cca-836a-0433a8da44ba/imagenet/dictionary.csv')
-    reader = csv.reader(codecs.iterdecode(response, 'utf-8'))
+    response = requests.get('https://storage.googleapis.com/tf-serving-docker-http-eae9e0c7-661d-4cca-836a-0433a8da44ba/imagenet/dictionary.csv')
+    reader = csv.reader(response.text.decode('utf-8').splitlines())
```
Comment (stephensolis) [inline tests/conftest.py]: #decode will fail in Python 3 but I guess we can fix similar issues in a future PR

**Pull request merged**